src/env.h

```diff
@@ -278,6 +280,7 @@ constexpr size_t kFsStatsBufferLength =
   V(netmask_string, "netmask")                                                 \
   V(next_string, "next")                                                       \
   V(nistcurve_string, "nistCurve")                                             \
+  V(node_string, "node")                                                       \
```

Comment:
node_string seems a bit generic here. (same with default and dot above)... although I after reviewing the entire change I'm seeing that this naming convention started before this PR. I do think it would be good to be more explicit such as `export_map_node_string`

Why is there no `require_string`?

Reply:
These are just a common string store in the C++ codebase, to avoid having to create constant strings inline each time they are used.

Reply:
fair enough... still think "node_string" sounds slightly more important, although I get that making it too verbose is probably not ideal.

Reply:
The naming pattern should always be along the lines of `X_string` for a literal string `X` – this line isn’t tied to the modules implementation in any way, if other parts of the C++ codebase ever need to use a JS string `'node'` then they could use this one too